node_scripts/install/spark.py

```diff
@@ -200,3 +200,28 @@ def setup_conf():
     except Exception as e:
         print("Failed to copy spark-defaults.conf file")
         print(e)
+
+
+    spark_default_path_src = os.path.join(os.environ['DOCKER_WORKING_DIR'], 'conf/core-site.xml')
```

Comment:
can you split this into separate method this setup_conf is getting quite long for more or less separate configuration